### ISSUE TITLE
Dropdown list on book/chapter/page view doesn't expand correctly

### ISSUE THREAD

**Issue by vasiliev123** [BookStackApp/BookStack#960]
**Describe the bug**
After clicking on the dropown page list (no matter which) in the book/chapter/page view, it collapses correctly. However when I click to expand the menu, sometimes it works and sometimes it expands and quickly collapses (see screenshots section, dropdown triangle under **Chapter 1** shows the list should be expanded but it isn't).
I checked only on 2 browsers:
 - Firefox 61.0.1 - it doesn't work correctly (ublocks etc. are disabled)
 - Chromium 68.0.3440.75 - it works correctly (no addons)

**Steps To Reproduce**
1. Go to book/chapter/page view
2. Click on page dropdown list few times (sometimes it buggs with the first click)
3. Voila!

**Expected behavior**
I would like to see expanded content :)

**Screenshots**
**Book view**
![book-view](https://user-images.githubusercontent.com/5650567/44024258-135616c8-9eed-11e8-89b6-0500ed435c4a.png)

**Page view**
![side-menu](https://user-images.githubusercontent.com/5650567/44024267-183ee426-9eed-11e8-9ee8-fcadd2c58727.png)

**Your Configuration (please complete the following information):**
 - Exact BookStack Version (Found in settings): 0.23.1
 - PHP Version: 7.2.8
 - Hosting Method (Nginx/Apache/Docker): Apache 2.4.6

**Comment by Kdecherf** [comment]
Confirmed here on 0.23.2. Could be related to the position of the opened page in the menu:

I'm seeing this bug when trying to expand the chapter containing the current opened page.

**Comment by Abijeet** [comment]
Can reproduce, will work on a fix.

**Comment by Abijeet** [comment]
Hi @vasiliev123,

Thanks for reporting this issue.

I was only able to reproduce this when clicking on these multiple times, and by double clicking them. This seems to be happening due to a race condition on quick mouse clicks.

**Comment by vasiliev123** [comment]
Well for I don't have to do it fast :)
![not-too-fast](https://user-images.githubusercontent.com/5650567/45609166-fb3f8d00-ba56-11e8-9dc3-fd7167130486.gif)

**Comment by Abijeet** [comment]
@vasiliev123 - Are you double clicking? This happens for me only if I click on the item again when its in the process of animation

**Comment by vasiliev123** [comment]
@Abijeet nope, these are single clicks every after about 2 seconds. As you can also see I've changed css a little bit (I've deleted the margins), of course I've double checked (about month ago) without any modifications :) problem still persisted. However I will make sure one more time tomorrow and post my results.

**Comment by ssddanbrown** [comment]
@Abijeet From my play-around this could occur easily as shown above when in Firefox & when the list was open by default (In the page sidebar). Your fix did seem to fix this though so have merged, Thanks. 
I've tried to make a little smoother and consistent in FireFox by animating max-height and delaying animation start but not sure I've made much difference. 

Overall, The main issue here should be fixed in the next release.
